docs/faq/README.md

```diff
@@ -0,0 +1,44 @@
+# Troubleshooting Common Issues
+
+### "Webhook not available" error when attempting to create a JobSet
+
+Example error message:
```

Comment:
Is this missing an error message?

I would wrap this in a code block maybe?

Reply:
Thanks, added the example error message.